integrationtests/src/test/java/io/kroxylicious/proxy/ExpositionIT.java

```diff
@@ -175,13 +173,9 @@ void exposesTwoSeparateUpstreamClustersUsingSniRouting(KafkaCluster cluster) thr
         try (var tester = kroxyliciousTester(builder)) {
             for (int i = 0; i < numberOfVirtualClusters; i++) {
                 var trust = keystoreTrustStoreList.get(i);
-                try (var admin = tester.admin("cluster" + i, Map.of(
-                        CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, SecurityProtocol.SSL.name,
-                        SslConfigs.SSL_TRUSTSTORE_LOCATION_CONFIG, trust.clientTrustStore(),
-                        SslConfigs.SSL_TRUSTSTORE_PASSWORD_CONFIG, trust.password()))) {
-                    // do some work to ensure virtual cluster is operational
-                    createTopic(admin, TOPIC + i, 1);
-                }
+                // do some work to ensure virtual cluster is operational
+                tester.createTopic("cluster" + i, Map.of(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, SecurityProtocol.SSL.name,
```

Comment:
probably not in scope for this PR, but I wonder if the tester should be smarter about this and figure out the TLS settings itself.